spark/src/main/scala/io/delta/sharing/spark/filters/OpConverter.scala

```diff
@@ -53,6 +53,8 @@ import org.apache.spark.sql.types.{
 // should skip filtering in such cases.
 object OpConverter {
 
+  val kMaxSqlInOpSizeLimit = 20
```

Comment:
Can you add some documentation to the definition of this limit?

Reply:
Done.